### ISSUE TITLE
Error while migrating to Swift 3

### ISSUE THREAD

**Issue by parthadrojasa** [mixpanel/mixpanel-swift#21]
I am calling `Mixpanel.mainInstance().track(event: name.rawValue, properties: dictionary)`, here error is dictionary as it is of type `[String: AnyObject]`.
So should i forcefully make it to Properties?
Also that gives me error like cannot convert value of type [String: AnyObject] to type Properties? in coercion. 

I am able to get rid of error if i change to [String: String]

**Comment by yarneo** [comment]
Hey @parthadrojasa thanks for reaching out. With the update of Swift3, we also made more string type enforcing on the properties. This means that the property values (which is AnyObject) in your example, must now conform to MixpanelType.
This is to assure you don't send unsupported properties. If you look in https://github.com/mixpanel/mixpanel-swift/blob/master/Mixpanel/MixpanelType.swift
you can see more about this.
MixpanelType can be either String, Int, UInt, Double, Float, Bool, [MixpanelType], [String: MixpanelType], Date, URL, or NSNull.

So in short, properties are now [String: MixpanelType]

**Comment by parthadrojasa** [comment]
@yarneo So if i want to create a dictionary which contains different type of values i just need to use [String: MixpanelType] in place of [String: AnyObject] Right?

**Comment by yarneo** [comment]
Yes that's correct !

**Comment by parthadrojasa** [comment]
@yarneo thanks for the prompt response.
